src/main/scala/uk/gov/nationalarchives/Lambda.scala

```diff
@@ -103,12 +112,16 @@ class Lambda extends RequestStreamHandler {
       _ <- folderUpdateRequests.map { folderUpdateRequest =>
         val message = generateTitleDescriptionSlackMessage(config.apiUrl, folderUpdateRequest)
         for {
+          _ <- log(s"Updating entity ${folderUpdateRequest.updateEntityRequest.ref}")
+          _ <- log(s"Sending\n$message\nto slack")
```

Comment:
Shouldn't this come after `entitiesClient.updateEntity(folderUpdateRequest.updateEntityRequest)`, just in case updating the entity fails?

Reply:
Good point, I've moved it.

Reply:
Oh no, I meant shouldn't the `log(s"Sending\n$message\nto slack")` message come after `entitiesClient.updateEntity(folderUpdateRequest.updateEntityRequest)` because how it is now, `entitiesClient.updateEntity(folderUpdateRequest.updateEntityRequest)` happens before the logging and  sending a message to slack is the first thing that is logged.

Reply:
Ah I see what you mean. I've put "updating entity" back and moved "sending to slack" down a bit.